cmd/insprd/operators/kafka/channels/client.go

```diff
@@ -155,3 +154,23 @@ func (c *ChannelOperator) Delete(ctx context.Context, context string, name strin
 	}
 	return nil
 }
+
+type kafkaAdminClient interface {
+	DeleteTopics(ctx context.Context, topics []string, options ...kafka.DeleteTopicsAdminOption) (result []kafka.TopicResult, err error)
+	CreateTopics(ctx context.Context, topics []kafka.TopicSpecification, options ...kafka.CreateTopicsAdminOption) (result []kafka.TopicResult, err error)
+	GetMetadata(topic *string, allTopics bool, timeoutMs int) (*kafka.Metadata, error)
+}
+
+type mockAdminClient struct {
+}
```

Comment:
Don't know if it's a good idea or not but it may be interesting to do a `err` field, in which the person using the mock can determine if they want an error or  not in the response.

Reply:
not really in the scope of your ticket/issue, probably will be done when the need for it comes